src/actions/curationActions.js

```diff
@@ -25,13 +26,31 @@ export function curateAction(token, spec) {
     dispatch(uiNotificationNew({ type: 'info', message: 'Started contribution.', timeout: 5000 }))
     return curate(token, spec).then(
       result => {
+        let url
```

Comment:
Consider replacing L29-35 with the following.

```javascript
const suffix = isDev ? '-dev' : ''
const url = `https://github.com/clearlydefined/curated-data${prefix}/pull/${result.prNumber}`
```
Not sure you can use process.env here. This code is not Node and is not running on the server. It is running in your browser. There was some funky **build time** stuff we did somewhere else (`app.js? www?`) to determine if we were in a dev environment. Should mimic that here. In fact, should likely put that in the Redux state so it just gets computed once and is available everywhere.

Reply:
I got the idea of using process.env.NODE_ENV from the registerServiceWorker.js and clearlyDefined.js files. In these files process.env.NODE_ENV is used to determine if the environment is development, test, or production. I tested it locally and it seems to be able to determine the correct environment (the temp variable in the code below was placed there temporarily to see what process.env.NODE_ENV is).
<img width="784" alt="screen shot 2018-06-20 at 9 08 44 am" src="https://user-images.githubusercontent.com/23349235/41670819-1fe94d72-746a-11e8-9a91-92df81aa20cd.png">

Reply:
not sure that will work in production. the code you reference in clearlyDefined.js only runs if `REACT_APP_SERVER` is not set. I believe it is. it is also true that NODE_ENV is not quite the way we expect. For example, on dev it is likely set ot production as it is a production execution environment whose purpose is to be the development environment. confusing but true.

Please validate. You'll need to check the build configurations as that env var is considered at build time.